File: MiniDOMWebLibs/MiniDOM/Uploads/Exception/UploadCalcelledException.vb
Imports minidom
Imports minidom.Databases
Imports minidom.Sistema
Imports System.Threading
Imports System.Web.UI.HtmlControls
Imports System.IO

Partial Class WebSite



    Public Class UploadCalcelledException
        Inherits System.Exception

        Public Sub New()
            DMDObject.IncreaseCounter(Me)
        End Sub

        Public Sub New(ByVal message As String)
            MyBase.New(message)
            DMDObject.IncreaseCounter(Me)
        End Sub

        Protected Overrides Sub Finalize()
            MyBase.Finalize()
            DMDObject.DecreaseCounter(Me)
        End Sub
    End Class


End Class

File: MiniDOMWebLibs1/MiniDOM/Uploads/Exception/UploadCalcelledException.cs
using System;

namespace minidom
{
    public partial class WebSite
    {
        public class UploadCalcelledException : Exception
        {
            public UploadCalcelledException()
            {
                DMDObject.IncreaseCounter(this);
            }

            public UploadCalcelledException(string message) : base(message)
            {
                DMDObject.IncreaseCounter(this);
            }

            ~UploadCalcelledException()
            {
                DMDObject.DecreaseCounter(this);
            }
        }
    }
}